### PR TITLE
Add assignments

### LINKED ISSUE #4: Aditya - HTML Basics Remake
@adithyaRocks Tag this when sending in your pull request.

### PR DESCRIPTION
**Pull request by AdithyaSuresh1501**
Fixes #4 

- All 3 assignments

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/48"></a>

### CODE REVIEW

File: Adithya/HTMLbasicsremake/form.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <label for="gender">Other:</label>
+        <input type="radio" name="gender" value="Other">
+        </p>
+        <p>
+        <label for="Email">Email:</label>
+        <input type="email" name="Email" placeholder="Email" required  >
+        <label for="Password">Password:</label>
+        <input type="password" name="Password" minlength="5" maxlength="15" required>
+        <label for="birthday">Birthday</label>
+        <input type="date" name="Birthday"><br>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `autocomplete`.

File: Adithya/HTMLbasicsremake/form.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <label for="Check">I agree to terms and conditions</label>
+        <input type="checkbox" name="Check" value="check"><br>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `required`.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Adithya/HTMLbasicsremake/Pokemon .html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <th><a href="http://pokemon.wikia.com/wiki/Wartortle">Wartortle</a></th>
+
+    
+</tr>    
+</table>    
+
+</body>
+
+
+</html>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/Pokemon .html:52]: Set indentation to tab 4 and format the file.

File: Adithya/HTMLbasicsremake/form.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Form</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form>
+        <p>
+        <label for="First Name">First Name</label>
+        <input type="text"  name="First Name" placeholder="First Name" required>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `autocomplete`.

File: Adithya/HTMLbasicsremake/form.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        <label for="Last Name">Last Name</label>
+        <input type="text"  name="Last Name" placeholder="Last Name" required>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `autocomplete`.

File: Adithya/HTMLbasicsremake/form.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        </p>
+        <p>
+        Gender:<label for="gender">Male:</label>
+        <input type="radio" name="gender" value="Male">
+        <label for="gender">Female:</label>
+        <input type="radio" name="gender" value="Female">
+        <label for="gender">Other:</label>
+        <input type="radio" name="gender" value="Other">
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `autocomplete`.

File: Adithya/HTMLbasicsremake/form.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    <h1>Register</h1>
+    <form>
+        <p>
+        <label for="First Name">First Name</label>
+        <input type="text"  name="First Name" placeholder="First Name" required>
+        <label for="Last Name">Last Name</label>
+        <input type="text"  name="Last Name" placeholder="Last Name" required>
+        </p>
+        <p>
+        Gender:<label for="gender">Male:</label>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: `label` not required here.

File: Adithya/HTMLbasicsremake/form.html [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        <input type="radio" name="gender" value="Male">
+        <label for="gender">Female:</label>
+        <input type="radio" name="gender" value="Female">
+        <label for="gender">Other:</label>
+        <input type="radio" name="gender" value="Other">
+        </p>
+        <p>
+        <label for="Email">Email:</label>
+        <input type="email" name="Email" placeholder="Email" required  >
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `autocomplete`.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by aravindvnair99** [comment]
@adithyaRocks Shall I close this as you have created another pull request from a separate branch?

**Comment by VinayakTekade** [comment]
@aravindvnair99 I have taken the other pull request into consideration, please close this pull request.

**Review by aravindvnair99 — COMMENTED** [review]
@adithyaRocks Merging and closing the pull request for now. Make the necessary changes in another pull request and send it over from a different branch. Also, make sure to fill up the pull request template and also refactor the code henceforth.

**Pull request merged**